### PR TITLE
slight docker improvements and removing gplus from person output

### LINKED ISSUE #27: gplus - remove functionality
GPlus is no longer, and has been removed from certain places in https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/273 . Thus, should be also removed from the scraper?

code:
https://github.com/JupiterBroadcasting/show-scraper/blob/5d0bbc996e90f260b003317fde0a131bdc52a644/models/person.py#L18

### PR DESCRIPTION
**Pull request by elreydetoda**
Started this because of:
https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/issues/265

Then did the docker optimizations because I was thinking of while testing for this PR :sweat_smile: 

closes #27

### CODE REVIEW

**Comment by gerbrent** [comment]
this seems like two very different PRs...  One for the `pipenv` additions, and another for the gplus change.

One I feel very comfortable merging, the other isn't my expertise... makes it a little tricky being combined!
So, someone else should have a look..

**Comment by elreydetoda** [comment]
I can split them up, I was just trying not to make too many small commits since it's been hard to get approvals in the repo overall.

**Comment by elreydetoda** [comment]
alright @gerbrent I've submitted #30 for the docker stuff and only have google plus stuff here. Thanks for letting me know that was your hesitancy :grin:

**Review by gerbrent — APPROVED** [review]

**Pull request merged**
